lib/chainweb/json.ex

```diff
@@ -0,0 +1,53 @@
+defprotocol Kadena.Chainweb.JSON do
```

Comment:
@santiagoboterokommit two comments:
1. since this is something related to PACT please, let's move it to `/chainweb/pact/...`. 
2. JSON is a general name and this parsing is specific to RequestBody modules, let's rename it as: `Kadena.Chainweb.Pact.JSONRequestBody`.